Dashboard/Views/ScrutineeringWindow.axaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Avalonia.Controls;
+using Avalonia.Interactivity;
+using Dashboard.ViewModels;
+
+namespace Dashboard.Views;
+
+public partial class ScrutineeringView : UserControl
```

Comment:
This doesnt compile after ur latest change, you need to update all references to use the new ScrutineeringView name now. You should probably leave it as is until Richard's ticket gets in (re UserControl etc)

Reply:
i will wait

Reply:
There was a couple things not changed to accommodate the UserControl change.

1. You'll need to update all references to use ScrutineeringView, the constructor wasn't updated and was still ScrutineeringWindow.
2. The Window.Resource part in the axaml would need to be UserControl.Resource as well.
3. The OnClosed method should be removed since it is no longer a Window and the MainWindowView will handle the dispose call.